src/main/java/qna/domain/Question.java

```diff
@@ -39,22 +41,24 @@ public Question(long id, String title, String contents) {
         this.contents = contents;
     }
 
-    public String getTitle() {
-        return title;
-    }
+    public DeleteHistories delete(User user) throws Exception {
 
-    public Question setTitle(String title) {
-        this.title = title;
-        return this;
+        DeleteHistories deleteHistories = new DeleteHistories();
+        deleteHistories.add(deleteQuestion(user));
+        new Answers(getAnswers()).delete(user, deleteHistories);
```

Comment:
Question의 인스턴스 필드로 private List<Answer> answers = new ArrayList<>();를 쓰지 않고 일급컬렉션을 사용하는건 어떨까요? 🙂 
new 연산자로 일급컬렉션을 만드는 로직은 delete 메서드에서 수행하기 적절하지 않은것 같습니다.
```suggestion
        answers.delete(user, deleteHistories);
```